pkg/omf/functions/packages/omf.packages.update.fish

```diff
@@ -1,11 +1,14 @@
-function omf.update -a name
-  function __omf.update.success
-    echo (omf::em)"✔ $argv successfully updated."(omf::off)
-  end
+function __omf.packages.update.success
+  echo (omf::em)"✔ $argv successfully updated."(omf::off)
+  return 0
```

Comment:
Everywhere else we have these helper functions we dont put the return value inside of the function.